docs/tools/vega-wallet/desktop-app/index.md

```diff
@@ -3,6 +3,9 @@ title: Desktop Wallet
 hide_title: false
 ---
 
-The Vega Wallet is available to use as a desktop application. It allows you to use your existing Vega wallet, create new wallets, or restore wallets. The desktop app also lets you connect to Vega networks and start Vega dApps through the visual interface.
```

Comment:
Suggestion (non-blocking): remove this line. I this doesn't need to act like news.